### DISCUSSION TITLE
Welcome

### DISCUSSION THREAD

**Discussion by jonathan-robertson** [jonathan-robertson/gmo-farming#53] [General]
If anyone has any questions about usage or contribution to this project, please feel free to post a comment here 😄

**Comment by sphinxlady** [comment]
Love the idea and makes a lot of sense for this type of game.

Any chance of a undead legacy version?

Think the only thing that would need change is the hot box to work with the subquake overhaul

**Reply by jonathan-robertson** [reply]
Hi! I don't have Undead Legacy set up right now to confirm, but is it giving you an error message?

**Reply by sphinxlady** [reply]
The crops do as they are suppose to in testing  but the issue seems to be the hot box itself.

There some issue with it and from testing once placed it randomly turns into different workstation.

Not so much a error it would seem but a incompatibility.

**Reply by jonathan-robertson** [reply]
Interesting.. do you encounter this issue on a brand new Undead Legacy save?
(one that's had GMO Farming installed from the very beginning)

**Reply by sphinxlady** [reply]
Started a new game and GMO is added to journal correctly on start.

Place hotbox and like before it randomly turns into different workstations from the undead legacy mod.

**Reply by jonathan-robertson** [reply]
Oh wow. Thanks for confirming. I thought that maybe the block IDs were borked or something (happens to me sometimes when testing new blocks in an existing world. 

Doesn't sound like that's the case here. Perhaps he's hot-swapping block IDs here like he does (or used to) for vehicles. 

Not really sure what I can do about this for the time being without knowing how Subquake's internals work, but I'll create an entry over in Issues as a reminder to look into it more when I have some extra time.

**Reply by jonathan-robertson** [reply]
Ref: https://github.com/jonathan-robertson/gmo-farming/issues/78
